lib/resty/upstream/healthcheck.lua

```diff
@@ -597,6 +605,16 @@ function _M.spawn_checker(opts)
         return nil, "no upstream specified"
     end
 
+    local hook_down = opts.hook_down
```

Comment:
Just trying to promote good practices since many people are inspired by the OpenResty official libraries to write their own ngx_lua code :)